app/main.py

```diff
@@ -1,6 +1,22 @@
 from typing import Callable
+import functools
+
+
+cached = {}
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    @functools.wraps(func)
+    def wrapper(*args) -> int:
+        global cached
+        key = str(id(func)) + "(" + f"{args}" + ")"
```

Comment:
There is ready to use function for this purpose `(args, frozenset(kwargs.items()))`